src/nodes/variable_declaration.js

```diff
@@ -30,10 +32,28 @@ export function format(node, context, recur) {
     }
 
     indent && context.indentIn();
+
     for (let i = 0; i < node.declarations.length; i++) {
-        recur(node.declarations[i]);
-        if (node.declarations[i + 1]) {
-            context.write(',\n', context.getIndent());
+        let previous = node.declarations[i - 1];
+        let child = node.declarations[i];
+        let next = node.declarations[i + 1];
+
+        context.write(blockComments.printLeading(child, previous, next));
+
+        if (i > 0) {
```

Comment:
cause the first one is already indented here https://github.com/dmitriiabramov/esfmt/pull/23/files#diff-5ecfbba207548f6ad3e168d2beddac6cR34